docs/azure_jumpstart_hcibox/using_hcibox/_index.md

```diff
@@ -0,0 +1,77 @@
+
+  ![Screenshot showing HCIBox nested virtualization stack diagram](./nested_virtualization_arch.png)
+
+| Computer Name    | Role                                | Domain Joined | Parent Host     | OS                  |
+| ---------------- | ----------------------------------- | ------------- | --------------- | ------------------- |
+| _HCIBox-Client_  | Primary host                        | No            | Azure           | Windows Server 2022 |
+| _AzSHOST1_       | HCI node                            | Yes           | _HCIBox-Client_ | Azure Stack HCI     |
+| _AzSHOST2_       | HCI node                            | Yes           | _HCIBox-Client_ | Azure Stack HCI     |
+| _AzSMGMT_        | Nested hypervisor                   | No            | _HCIBox-Client_ | Windows Server 2022 |
+| _JumpstartDC_    | Domain controller                   | Yes (DC)      | _AzSMGMT_       | Windows Server 2022 |
+| _Vm-Router_     | Remote Access Server                | No            | _AzSMGMT_       | Windows Server 2022 |
+
+### Active Directory domain user credentials
+
+Once you are logged into the _HCIBox-Client_ VM using the local admin credentials you supplied in your template parameters during deployment you will need to switch to using a domain account to access most other functions, such as logging into the HCI nodes or accessing Windows Admin Center. The default domain account is _administrator@jumpstart.local_.
+
+  > **Note:** The password for this account is set as the same password you supplied during deployment for the local account. Many HCIBox operations will use the domain account wherever credentials are required.
+
+### Monitoring Azure Stack HCI
+
+Azure Stack HCI integrates with [Azure Monitor](https://learn.microsoft.com/azure-stack/hci/manage/monitor-hci-single) to support monitoring HCI cluster insights through the Azure portal. Follow these steps to configure monitoring on your HCIBox cluster.
+
+- Open the Overview blade of your cluster and see that platform monitoring is available by default.
+
+  ![Screenshot showing platform monitoring in Az monitor](./monitor_platform.png)
+
+- From the Overview blade of the _HCIBox-Cluster_ resource, select the "Capabilities" tab, then click on Insights and then "Getting Started." Follow the wizard to create a new data collection rule and data collection endpoint.
+
+  ![Screenshot showing capabilities tab](./create_dcr.png)
+
+- It will take time for logs data to flow through to Insights. Once data is available, click on the Insights blade of the _HCIBox-Cluster_ resource to view the Insights workbook and explore logs from your cluster.
+
+  ![Screenshot showing logs configured](./hci_insights.png)
```

Comment:
```suggestion
### Monitoring Azure Stack HCI

Azure Stack HCI integrates with [Azure Monitor](https://learn.microsoft.com/azure-stack/hci/manage/monitor-hci-single) to support monitoring HCI cluster insights through the Azure portal. Follow these steps to configure monitoring on your HCIBox cluster.

- Open the Overview blade of your cluster and see that platform monitoring is available by default.

  ![Screenshot showing platform monitoring in Az monitor](./monitor_platform.png)

- From the Overview blade of the _HCIBox-Cluster_ resource, select the "Capabilities" tab, then click on Insights and then "Getting Started." Follow the wizard to create a new data collection rule and data collection endpoint.

  ![Screenshot showing capabilities tab](./create_dcr.png)

- It will take time for the logs data to flow through to Insights. Once data is available, click on the Insights blade of the _HCIBox-Cluster_ resource to view the Insights workbook and explore logs from your cluster.

  ![Screenshot showing logs configured](./hci_insights.png)
```